src/hooks/useConcentratorTokenPriceUsd.ts

```diff
@@ -0,0 +1,41 @@
+import { useQuery } from "@tanstack/react-query"
+import { Address } from "wagmi"
+
+import { getGlpPrice } from "@/lib/api/pricer/getGlpPrice"
+import { getLlamaPrice } from "@/lib/api/pricer/getLlamaPrice"
+import { queryKeys } from "@/lib/helpers"
+import { useActiveChainId, useIsConcentratorTokenVault } from "@/hooks"
+
+export function useConcentratorTokenPriceUsd({
```

Comment:
I don't understand why this is needed. All tokens should be able to be priced with one hook, no?

On the surface this looks very similar to `useTokenPriceUsd`, just expressed differently.

Reply:
You're right, don't remember exactly why I split the concentrator and the compounder token price USD. But updating the customPricers should be ok/enough.